tests/test_sign_cmd.py

```diff
@@ -1,17 +1,23 @@
 from hashlib import sha256
 from sha3 import keccak_256
+from pathlib import Path
 
 from ecdsa.curves import SECP256k1
 from ecdsa.keys import VerifyingKey
 from ecdsa.util import sigdecode_der
 
 from boilerplate_client.transaction import Transaction
+from boilerplate_client.boilerplate_cmd import BoilerplateCommand
+from ragger.navigator import NavInsID, NavIns
```

Comment:
no used
```suggestion
```

Reply:
It will be used very soon when stax nav is added